.infra/ansible/roles/testnet/tasks/testnet.yaml

```diff
@@ -0,0 +1,190 @@
+- name: Make base directory for testnet dependencies | testnet
+  file:
+    path: "/opt/testnet"
+    mode: 0755
+    state: directory
+    recurse: true
+
+- name: Create service network | tesnet
+  docker_network:
+    name: testnet
+
+- name: Login to registry containing testnet image | testnet
+  command: "docker login -u {{ docker_testnet_pull_token_login }} -p {{ docker_testnet_pull_token_password }} {{ testnet_image }}"
+
+# Data for every node should be saved in subdirectory for a future sync
+- name: Make base directory for testnet node files | testnet
+  file:
+    path: "/opt/testnet/{{ inventory_hostname }}"
+    mode: 0755
+    state: directory
+    recurse: true
+
+- name: Bootstrap first node | testnet
+  run_once: true
+  block:
+    - name: Run node0 prebootstrap commands | testnet-bootstrap
+      command: |
+        docker run -i --name testnet_preinit -v /opt/testnet/{{ inventory_hostname }}:/root/ {{ testnet_image }} bash -c "
+        dkgcli gen-key-file {{ herb_threshold_2 }} {{ testnet_clients_amount }};
+        "
+      args:
+        creates: "/opt/testnet/{{ inventory_hostname }}/.dkgcli/keys.json"
+
+    - name: Save prebootstrap logs | testnet-bootstrap
+      shell: docker logs testnet_preinit > /opt/testnet/prebootstrap.log
+      args:
+        creates: "/opt/testnet/prebootstrap.log"
+
+    - name: Fetch prebootstrap logs | testnet
+      fetch:
+        src: "/opt/testnet/prebootstrap.log"
+        dest: "fetch/prebootstrap.log"
+        flat: true
+
+    - name: Fetch prebootstrap keys | testnet
+      fetch:
+        src: "/opt/testnet/{{ inventory_hostname }}/.dkgcli/keys.json"
+        dest: "fetch/dkgcli_keys.json"
+        flat: true
+
+    - name: Get dkgcli_keys.json as variable | testnet-bootstrap
+      include_vars:
+        name: "dkgcli_keys"
+        file: "fetch/dkgcli_keys.json"
+
+    - name: Run node0 bootstrap commands | testnet-bootstrap
+      command: |
+        docker run -i --name testnet_init -v /opt/testnet/{{ inventory_hostname }}:/root/ {{ testnet_image }} bash -c "
+        hd init {{ inventory_hostname }} --chain-id {{ env_name }};
+
+        for i in $(seq 1 {{ testnet_clients_amount }}); do
+          hcli keys add user$i <<< '{{ testnet_client_password }}';
+        done;
+        hcli keys add sellerBeneficiary <<< '{{ testnet_client_password }}';
```

Comment:
`hcli keys add sellerBeneficiary <<< '{{ testnet_client_password }}';`
`hcli keys add buyerBeneficiary <<< '{{ testnet_client_password }}';`
`hcli keys add dgaming <<< '{{ testnet_client_password }}';`
I suppose these accounts shouldn't be here. It is the marketplace settings.

Reply:
Fixed